rust/ql/src/queries/unusedentities/UnreachableCode.ql

```diff
@@ -29,7 +29,10 @@ predicate hiddenNode(AstNode n) {
   not succ(n, _) and
   not succ(_, n)
   or
-  n instanceof ControlFlowGraphImpl::PostOrderTree // location is counter-intuitive
+  n instanceof ControlFlowGraphImpl::PostOrderTree and // location is counter-intuitive
+  not n instanceof MacroExpr
```

Comment:
What's the difference between a `MacroExpr` and a `MacroCall`?

Reply:
A `MacroCall` can happen outside of expressions; a `MacroExpr` is a subset of those calls that happen in expression contexts.

Reply:
Ah, great, thanks!